docs/Documentation-CHN/UserGuide/2-Concept Key Concepts and Terminology/1-Key Concepts and Terminology.md

```diff
@@ -166,6 +170,42 @@ IoTDB在显示时间戳时可以支持LONG类型以及DATETIME-DISPLAY类型，
 
 </center>
 
+* 相对时间戳
+  
+  相对时间是指与服务器时间```now()```和```DATETIME```类型时间相差一定时间间隔的时间。
+  形式化定义为：
+  ```
+  Duration = (Digit+ ('Y'|'MO'|'W'|'D'|'H'|'M'|'S'|'MS'|'US'|'NS'))+
+  RelativeTime = (now() | DATETIME) ((+|-) Duration)+
```

Comment:
I looked at the realization and the realization seems to also allow `Duration ((+|-) Duration)+`. What do you think?

Reply:
5Y6Y is equal to 11Y. 7D8D is equal to 7D+8D, which is 15D.

Reply:
oh, it indeed allows Duration ((+|-) Duration)+. I will fix it.